### ISSUE TITLE
Capability to determine make image behavior due to sysprep.

### ISSUE THREAD

**Issue by binod33** [dasein-cloud/dasein-cloud-core#6]
Some clouds like azure does not run sysprep on VMs automatically and requests the users to verify if the  sysprep has been run or not before making an image from that particular VM. If dasein can tell users which clouds does not run sysprep automatically or provides the verification option via capability, it might easier for the user to be aware of these cases before imaging a vm.

**Comment by drewlyall** [comment]
I don't believe there are any clouds that automatically run sysprep when imaging the VM. Obviously Azure does not but neither does AWS or GCE or vSphere.
I'm not exactly sure what capability we would be putting in place here?

**Comment by binod33** [comment]
AWS generates new windows vm passwords as a default behavior even when launching from a non sysprepped image. But in case of Azure, sysprepped images is necessary in order for the user provided credentials to be set in the vm during launch. So maybe a capability to describe such behavior in cloud like 
settingBoopstrapUserRequiresSyspreppedImage

**Comment by drewlyall** [comment]
That's getting into awfully hypervisor level stuff, not really cloud level.
Dasein shouldn't really know nor care whether you're sysprepping an image.
And AWS specifically recommends that you do...
